Client/Frontend/Browser/Tabs/Legacy/LegacyTabCell.swift

```diff
@@ -226,7 +226,7 @@ class LegacyTabCell: UICollectionViewCell,
         screenshotView.image = nil
         backgroundHolder.transform = .identity
         backgroundHolder.alpha = 1
-        self.titleText.font = LegacyDynamicFontHelper.defaultHelper.DefaultSmallFontBold
+        self.titleText.font = DefaultDynamicFontHelper.preferredFont(withTextStyle:.body,size:12,weight:.semibold)
```

Comment:
There should be spaces between params, we have Swiftlint enabled in the project and CI pipeline and this will trigger warnings otherwise.
```suggestion
        self.titleText.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body, size:12, weight: .semibold)
```